readme.md

```diff
@@ -37,6 +37,8 @@ docker-compose exec fpm bash
 composer install # this is inside container
 cp .env.example .env
 php artisan key:generate # copy and paste this key in the APP_KEY section of the .env file
+php artisan migrate
+php artisan passport:install # copy credentials into .env at PERSONAL_CLIENT_SECRET and PASSWORD_CLIENT_SECRET
 ```
```

Comment:
Note for the futur automate this in the container ?

Reply:
Yeah ideally I would like to make an installer in the future (graphic one) that would make all of this to avoid any errors from PEBKAC